src/EdFi.AnalyticsMiddleTier.DataStandard31/Asmt/MSSQL/0011-View-AssessmentFact-Create.sql

```diff
@@ -0,0 +1,76 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+IF EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.VIEWS WHERE TABLE_SCHEMA = 'analytics' AND TABLE_NAME = 'asmt_AssessmentFact')
+BEGIN
+	DROP VIEW analytics.asmt_AssessmentFact
+END
+GO
+
+CREATE VIEW analytics.asmt_AssessmentFact
+AS
+	SELECT
+       CONCAT(Assessment.AssessmentIdentifier, '-', Assessment.Namespace)  AS AssessmentKey,
+       Assessment.Namespace,
+       Assessment.AssessmentTitle AS Title,
+       COALESCE(Assessment.AssessmentVersion, 0) AS Version,
+       COALESCE(CategoryDescriptor.Description, '') AS Category,
+       Descriptor.Description AS AssessedGradeLevel,
+       COALESCE(AcademicSubjectDescriptor.Description,'') AS AcademicSubject,
+       CONCAT(ObjectiveAssessment.AssessmentIdentifier, '-', ObjectiveAssessment.IdentificationCode, '-', ObjectiveAssessment.Namespace) AS ObjectiveAssessmentKey,
+       ObjectiveAssessment.IdentificationCode,
+       CASE WHEN ObjectiveAssessment.ParentIdentificationCode IS NOT NULL
+            THEN CONCAT(ParentAssesment.AssessmentIdentifier, '-', ParentAssesment.IdentificationCode, '-', ParentAssesment.Namespace)
+          ELSE ''
+          END AS ParentObjectiveAssessmentKey,
+       COALESCE(ObjectiveAssessment.Description, '') as ObjectiveAssessmentDescription,
+       COALESCE(ObjectiveAssessment.PercentOfAssessment,0) as PercentOfAssessment,
+       COALESCE(AssessmentScore.MinimumScore, ObjectiveAssessmentScore.MinimumScore, '') as MinScore,
+       COALESCE(AssessmentScore.MaximumScore, ObjectiveAssessmentScore.MaximumScore, '') as MaxScore
```

Comment:
The MinScore and MaxScore tests do not consider that the field is null, if the COALESCE is changed the tests do not fail.